python/hyperon/exts/snet_io/snet_gate.py

```diff
@@ -88,6 +88,8 @@ def __call__(self, command_a, *args_a):
             self.init_sdk()
         if command == 'get_service_callers':
             return args[0].generate_callers()
+        if command == 'get_service_callers_text':
```

Comment:
there was no need to expose this function to metta

Reply:
Ok, i removed those lines.